docker/service/nginx-v1/image/files/etc/nginx/koi-utf

```diff
@@ -0,0 +1,109 @@
+
+# This map is not a full koi8-r <> utf8 map: it does not contain
```

Comment:
@212223 are these files really needed? why cannot we stick to simpler nginx alpine variant? wdyt?

Reply:
Yes we can, now nginx-v1 dir has content:

└── [dir]  image
    ├── [602]  Dockerfile
    └── [dir]  files
        └── [dir]  etc
            └── [dir]  nginx
                └── [dir]  conf.d
                    └── [894]  default.conf

It is based on the official repository [nginx](https://hub.docker.com/_/nginx)